api/api-doc/storage_service.json

```diff
@@ -3546,6 +3546,24 @@
                "description":"The resulting compression ratio (estimated on a random sample of files)"
             }
          }
+      },
+      "sstable_to_restore":{
+         "id":"sstable_to_restore",
+         "description":"SSTable to restore and its token range",
+         "properties":{
+            "toc_filename":{
+               "type":"string",
+               "description":"the TOC filename of the sstable"
+            },
+            "token_min":{
+               "type":"string",
+               "description":"the minimum token found in the sstable"
+            },
+            "token_max":{
+               "type":"string",
+               "description":"the maximum token found in the sstable"
```

Comment:
Can we make token_min and token_max optional?
In their absence, we can either assume the whole token range and use mutation based streaming, or read the SSTable's Summary file to get the min/max keys and calculate the min/max tokens from that (given the schema).
Scylla manager may not have this metadata when restoring from an old backup that didn't have https://github.com/scylladb/scylladb/issues/26352 and https://github.com/scylladb/scylla-manager/issues/4582